spec/lib/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,10 @@
+## An example test to demonstrate that tests were setup properly
+## To be removed when first tests are written
+
+RSpec.describe "tic-tac-toe" do
+  context "make sure tests are working" do
```

Comment:
Very small comment, especially on a trivial test case, but the convention for contexts is to begin with `when` so that your contexts and descriptions flow together like a sentence. e.g. 'when the tests are working it should pass'

Reply:
that makes a lot of sense! thanks!